### ISSUE TITLE
Cinnamon crashes while moving icons in the panel

### ISSUE THREAD

**Issue by atloykenji** [linuxmint/cinnamon#10724]
```
 * Cinnamon 5.2.7
 * Linux Mint 20.3 Una
 * Kernel 5.4.0-104-generic
 * Nvidia Geforce GTX960 with nvidia-driver-470
 * 64 bit
 ```

I have sporadic freezes while moving panel icon of application, only the mouse pointer can move. The search menu worked. I can open new applications by search through it, but can't close them because other things on desktop is unresponsive. Programs keep running, music and media keeps playing… Ctrl+Alt+F2 worked, and I did manage to fix this by switching to CLI mode and reboot the system.
-Thanks for fixing this issue in advance!

**Comment by cirnod** [comment]
I observed the same issue. Restarting Cinnamon from command line via Ctrl+Alt+F2 also "fixed" it in my cases.

System info:
```
OS: Linux Mint 20.3 Cinnamon
Cinnamon: 5.2.7
Kernel: 5.4.0-100-generic
Processor: Intel© Core™ i7-6600U CPU @ 2.60GHz × 2
Graphics card: Intel Corporation Skylake GT2 [HD Graphics 520]
```

**Comment by splitowo** [comment]
Duplicate #10618 
The fix is merged and to be released in cinnamon 5.4.0 (Mint 21)
